Description: Don't build nim-lang.org website
 This is unnecessary to install on users' machines.
Author: Ximin Luo <infinity0@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tools/nimweb.nim
+++ b/tools/nimweb.nim
@@ -440,7 +440,8 @@
   buildNewsRss(c, "web/upload")
 
 proc main(c: var TConfigData) =
-  buildWebsite(c)
+  if not existsDir("web/upload"):
+    createDir("web/upload")
   buildJS("web/upload")
   buildAddDoc(c, "web/upload")
   buildDocSamples(c, "web/upload")
